releases/unreleased/set-sortinghat-as-a-required-package-in-elk.yml

```
@@ -0,0 +1,9 @@
+---
+title: Set SortingHat as a required package in elk
```

I think this title is better for the changelog.

```suggestion
title: SortingHat as a required package
```